### PR TITLE
[1LP][RFR] V2V CSV test

### PR DESCRIPTION
**Pull request by sshveta**
{{ pytest: cfme/tests/v2v/test_csv_import.py --use-provider rhv42 --use-provider vsphere67-nested --provider-limit 2 -vvvv --long-running}}

### CODE REVIEW

File: cfme/tests/v2v/test_csv_import.py [hunk 1]
```diff
@@ -16,44 +15,48 @@
...
+    plan_obj = appliance.collections.v2v_migration_plans
     view = navigate_to(plan_obj, 'Add')
     view.general.fill({
-        'infra_map': infra_map.name,
-        'name': plan_name,
-        'description': fauxfactory.gen_alpha(20)
+        "infra_map": infra_map.name,
+        "name": fauxfactory.gen_alpha(10),
+        "description": fauxfactory.gen_alpha(10),
+        "select_vm": radio_btn
```
Comment (john-dupuy) [inline cfme/tests/v2v/test_csv_import.py]: Q: Would it be possible to have the import be a separate widget from `select_vm`? It seems odd to fill "select_vm" with radio_btn

**Comment by john-dupuy** [comment]
Also were these tests run locally? Are v2v tests not run on PRT?

File: cfme/tests/v2v/test_csv_import.py (hunk 1 above)
Comment (sshveta) [reply cfme/tests/v2v/test_csv_import.py]: Changed it to choose_vm . They are two radio buttons to choose which way the Vm's will be imported .

File: cfme/tests/v2v/test_csv_import.py [hunk 2]
```diff
@@ -16,44 +15,48 @@
...
+    mapping = infrastructure_mapping_collection.create(**infra_mapping_data)
+    yield mapping
+    infrastructure_mapping_collection.delete(mapping)
 
 
 def migration_plan(appliance, infra_map, csv=False):
     """Function to create migration plan and select csv import option"""
-    plan_name = "map_{}".format(fauxfactory.gen_alpha(10))
-    plan_obj = appliance.collections.v2v_plans
+    radio_btn = "Import a CSV file with a list of VMs to be migrated"
```
[the remark this answers is not in the document]
Comment (sshveta) [reply cfme/tests/v2v/test_csv_import.py]: Done

**Comment by sshveta** [comment]
> Also were these tests run locally? Are v2v tests not run on PRT?

5.10.4 has an issue with conversion host . So tests are skipping . 
Test run and pass on 5.10.3

File: cfme/tests/v2v/test_csv_import.py [hunk 3]
```diff
@@ -16,44 +15,48 @@
...
-        view.general.select_vm.select("Import a CSV file with a list of VMs to be migrated")
-        view.next_btn.click()
+
+    view.next_btn.click()
     return view
 
 
-def import_and_check(appliance, infra_map, error_text, filetype='csv', content=False,
-                     table_hover=False, alert=False):
+def import_and_check(appliance, infra_map, error_text=None, csv=False,
```
Comment (digitronik) [inline cfme/tests/v2v/test_csv_import.py]: same as above why `csv`?

**Comment by sshveta** [comment]
Teardown of provider is failing in 5.11 because of BZ https://bugzilla.redhat.com/show_bug.cgi?id=1703744

**Pull request merged**